libbpfgo.go

```diff
@@ -672,7 +661,7 @@ func (b *BPFMap) UpdateBatch(keys, values unsafe.Pointer, count uint32) error {
 	}
 	errC := C.bpf_map_update_batch(b.fd, keys, values, &countC, bpfMapBatchOptsToC(&opts))
 	if errC != 0 {
-		return fmt.Errorf("failed to update map %s: %v", b.name, errC)
+		return fmt.Errorf("failed to update map %s: %w", b.name, syscall.Errno(int(errC)))
```

Comment:
I think we should wrap the errno value directly, due to inconsistencies with the returned value and errno in libbpf.